### PR TITLE
CALS-5698 Other adults submit impl 

### PR DESCRIPTION
**Pull request by lauherkcambria**
#### Which User story does this relate to?
- https://osi-cwds.atlassian.net/browse/CALS-5698

#### Brief feature description
-  add validations to other adults card for submit

#### Does this PR block/effect other stories?
- no

#### Do any commands needs to be run in order to run the project with this PR?
- no

#### Code coverage percent diff as a result of this PR?
- 
85.39% Statements 
76.29% Branches 
81.52% Functions
85.82% Lines

### CODE REVIEW

**Comment by ratneshraval** [comment]
@lauherkcambria same comments here as #408 
1. Move `this.isRelationshipToApplicantObject` to top of render to evaluate once and use throughout render. 
2. Add some unit tests for validator since the use is growing fast.

**Comment by lauherkcambria** [comment]
jenkins, retest this please.

**Comment by lauherkcambria** [comment]
@ratneshraval I have created technical debt story https://osi-cwds.atlassian.net/browse/CALS-5879 to cover bringing the code coverage up as discussed.

File: app/javascript/components/common/OtherAdultsCardField.js [hunk 1]
```diff
@@ -12,44 +13,97 @@ const dateValidator = {rule: 'isValidDate', message: 'date is invalid'}
...
-  render () {
-    const adult = this.props.otherAdults
+  isRelationshipToApplicantObject () {
+    const val = this.props.otherAdults.relationship_to_applicants[0].relationship_to_applicant
+    return _.isObject(val)
+  }
 
-    const otherAdultsRuleId = this.props.idPrefix + 'date_of_birth'
+  componentWillUnmount () {
+    const rulesToRemove = [ this.otherAdultDOBId, this.relationshipToApplicantID, this.ApplicantIdID,
```
Comment (ratneshraval) [inline app/javascript/components/common/OtherAdultsCardField.js]: Can you add test cases to verify addition and removal of rules?

**Pull request merged**